config/holes.toml

```
@@ -1412,8 +1412,11 @@ category = 'Transform'
 synopsis = 'Tally code golf medals.'
 preamble = '''
 <p>
-    Given a list of space separated scores in ascending order, print the total
-    number of diamonds and medals handed out.
+    Solutions on code.golf can be awarded one of three medals: bronze (🥉) for 3rd place, silver (🥈) for 2nd place, or gold (🥇) for 1st place.
+    In addition, an uncontested 1st place solution is awarded a diamond (💎). Tied solutions are all awarded the same medal.
+
+<p>
+    Given a list of space-separated solution scores in ascending order, print the number of diamonds and medals handed out.
```

```suggestion
    Given a list of space-separated solution scores in ascending order, print the applicable number of diamonds and medals.
```
rephrased to use active voice. feel free to discard

---

I appreciate the shift to active voice, though I don't like the word "applicable" here.

---

maybe "appropriate". or maybe "print the corresponding diamonds and medals"